crates/ruff_linter/src/rules/refurb/rules/slice_to_remove_prefix_or_suffix.rs

```diff
@@ -248,6 +248,30 @@ fn affix_removal_data<'a>(
         return None;
     }
     let slice = slice.as_slice_expr()?;
+
+    // Exit early if slice step is...
+    if slice
+        .step
+        .as_ref()
+        // present and
+        .is_some_and(|step| match step.as_ref() {
+            // not equal to 1
+            ast::Expr::NumberLiteral(ast::ExprNumberLiteral {
+                range: _,
+                value: ast::Number::Int(x),
+            }) => !(x.as_u8().is_some_and(|val| val == 1)),
```

Comment:
I think this should work:

```suggestion
            }) => x.as_u8() != Some(1),
```

Reply:
much better! Done.